docs/contribute/install.mdx

```diff
@@ -85,6 +85,15 @@ mindsdb: http API: started on 47334
 
 You can access the MindsDB Editor at `localhost:47334`.
 
+## How to Overcome `ImportError: failed to find libmagic`
```

Comment:
Let's change this to <Tip> instead of new Header

Reply:
what do you mean by new Header?

Reply:
@maulanaakbardj 
Please replace it with the below:

> \<Tip>
> If you get the \`ImportError: failed to find libmagic\` error, you should install the \`libmagic\` manually by running one of the commands below:
> 
> \```bash
> pip install python-magic-bin  # for linux and windows
> brew install libmagic  # for macOS
> \```
> \</Tip>

Reply:
done @ZoranPandovski @martyna-mindsdb 

![image](https://github.com/mindsdb/mindsdb/assets/67249292/686ded89-1d7c-41c1-ac23-6691abe4f4a9)